sys-utils/lsns.c

```diff
@@ -278,6 +278,26 @@ static int column_name_to_id(const char *name, size_t namesz)
 	return -1;
 }
 
+static int is_column_in_filter(int id, struct libscols_filter *filter)
+{
+	int r = 0;
+	const char *name = NULL;
+	struct libscols_iter *itr = scols_new_iter(SCOLS_ITER_FORWARD);
+
+	if (!itr)
+		err(EXIT_FAILURE, _("failed to allocate iterator"));
+
+	while (scols_filter_next_holder(filter, itr, &name, 0) == 0) {
+		if (strcmp(infos[id].name, name) == 0) {
+			r = 1;
+			break;
+		}
+	}
+
+	scols_free_iter(itr);
+	return r;
+}
+
```

Comment:
I have added [scols_filter_has_holder()](https://github.com/util-linux/util-linux/pull/3454/commits/8097dd31c24cb7f10d404ec5f756f9f5a7c248a1) to avoid this.